src/utils/user.ts

```diff
@@ -26,17 +26,13 @@ export const getUserLocationData = ({
   const userLocation: WithId<UserLocation> | undefined =
     worldUserLocationsById[user.id];
 
-  const userLastSeenIn =
-    Object.keys(userLocation.lastSeenIn)[0] &&
-    !!Object.keys(userLocation.lastSeenIn)[0].includes(roomName);
-  const userLastSeenLocation =
-    userLocation.lastSeenIn?.[venueName] ||
-    userLocation.lastSeenIn?.[roomName] ||
-    userLocation.lastSeenIn?.[location] ||
-    Object.values(userLocation.lastSeenIn)[0];
-  const isLocationMatch = userLastSeenIn && userLastSeenLocation;
+  const isLocationMatch =
```

Comment:
Doesn't this need to be modified like https://github.com/sparkletown/sparkle/pull/2202/files#diff-69b65f1189b0befb788170375aded3d463c6398d8804305bab5072c7624a5e93R44 ?

Reply:
Good catch. Done in https://github.com/sparkletown/sparkle/pull/2202/commits/c10b9c5940ef0d49677132d8549ebe515c2217e3